File: oldSnacks/python/creditcardvalidator.py
import creditcardvalidatorfunctions

credit_card_number = int(input("Enter your Card Numbers: "))
try:
	sum1 = creditcardvalidatorfunctions.get_sum_of_odd_postioned_numbers(credit_card_number)
	sum2 = creditcardvalidatorfunctions.get_sum_of_even_postioned_numbers(credit_card_number)

	creditcardvalidatorfunctions.get_issuer(credit_card_number)
	print("**Credit Card Number:",credit_card_number)
	print("**Credit Card Digit Length:", len(str(credit_card_number)))
	print("**Credit Card Valid:",creditcardvalidatorfunctions.get_validity(sum1, sum2))

except TypeError:

	print("INVALID DIGIT LENGTH!")

File: oldSnacks/Java/CreditCardValidator.java
import java.util.Scanner;
import java.util.Arrays;

public class CreditCardValidator{
public static void main(String...args){
	Scanner input = new Scanner(System.in);

	System.out.print("ENTER THE DIGITS LENGTH: ");
	int digitLength = input.nextInt();

	int [] digits = CreditCardValidatorFunctions.getCardNumberAsArray(digitLength);
	int sum1 = CreditCardValidatorFunctions.getSumOfOddPositionedDigits(digits);
	int sum2 = CreditCardValidatorFunctions.getSumOfEvenPositionedDigits(digits);
	System.out.println("\n\n*********************************************");
	CreditCardValidatorFunctions.getCardIssuer(digits);
	System.out.print("**Credit Card Number: ");
	CreditCardValidatorFunctions.printCardNumber(digits);
	System.out.println("\n**Credit Card Digit Length: "+digitLength);
	CreditCardValidatorFunctions.checkValidity(sum1, sum2);
	System.out.print("*********************************************");
}
}